templates/statefulset.yaml

```diff
@@ -363,9 +363,6 @@ spec:
 {{- if .Values.Lotus.enabled }}          
     - metadata:
         name: vol-lotus
-        {{- range $key, $value := .Values.persistence.lotus.annotations }}
```

Comment:
Why do we delete this?

Reply:
@ArseniiPetrovich  How do we use .Values.persistence.lotus.annotation? I didn't find any places and decided this part is useless.

Reply:
Not sure, but if you are sure it not used anywhere - I'm fine with the update.